providers/base/units/stress/s3s4.pxu

```diff
@@ -150,3 +150,221 @@ requires: cpuinfo.platform in ("i386", "x86_64")
 command: [ -e "$PLAINBOX_SESSION_SHARE"/hibernate_{s4_iterations}_cycles.log ] && cat "$PLAINBOX_SESSION_SHARE"/hibernate_{s4_iterations}_cycles.log
 _description:
  Attaches the log from the {s4_iterations} cycles Hibernate/Resume test if it exists
+
+id: stress_s3_cycles_iterations_1
+category_id: stress-tests/suspend
+summary:
+  Resource job meant to be used in the S3 cycles stress templates
+plugin: resource
+environ: STRESS_S3_ITERATIONS STRESS_SUSPEND_REBOOT_ITERATIONS
+command:
+ REBOOT_ITERATIONS=${STRESS_SUSPEND_REBOOT_ITERATIONS:-3}
+ for i in $(seq 2 "$REBOOT_ITERATIONS"); do
+     echo "suspend_reboot_id: $i"
+     echo "suspend_reboot_previous: $((i-1))"
+     python3 -c 'import platform;print("fwts: {}".format("supported" if platform.machine() in ["x86_64", "i386"] else "unsupported"))'
```

Comment:
I understand that you don't want to have two groups of jobs depending on the fwts support, so this is why you don't use the `requires (...)` here. But this can be made much simpler. The platform.machine() underneath goes back to the Unixy "uname".
See here: https://github.com/python/cpython/blob/main/Lib/platform.py#L908

So what would be much cleaner and easier (and faster! - as it doesn't invoke a new interpreter session) is:
```bash
architecture=$(uname -m)
if [ "$architecture" = "x86_64" ] || [ "$architecture" = "i386" ]; then
    echo "fwts: supported"
else
    echo "fwts: unsupported"
fi